### PR TITLE
Fix a typo

### PR DESCRIPTION
**Pull request by xchs**
Fix a typo in the application labels.

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
     "ui.server.contao.ltsTitle": "Long Term Support",
     "ui.server.contao.ltsText": "The LTS version has an extended support period of up to four years.",
     "ui.server.contao.latestTitle": "Latest",
-    "ui.server.contao.latestText": "A new latest version is release every six months in August and February.",
+    "ui.server.contao.latestText": "A new latest version is released every six months in August and February.",
```
Comment (xchs) [inline src/i18n/en.json]: Should we perhaps write "February and August" here?

File: src/components/App.vue [hunk 2]
```diff
@@ -2,7 +2,7 @@
     <div id="app">
         <div class="https-warning" v-if="isInsecure">
             <strong class="https-warning__headline">{{ 'ui.app.httpsHeadline' | translate }}</strong>
-            <span class="https-warning__description">{{ 'ui.app.httpsDescription' | translate }}</span>
+            <span class="https-warning__description"> {{ 'ui.app.httpsDescription' | translate }} </span>
```
Comment (aschempp) [inline src/components/App.vue]: Why did you add these whitespaces?

File: src/i18n/en.json [hunk 3]
```diff
@@ -186,7 +186,7 @@
     "ui.package.hintConstraintUpdate": "This package will be updated when you apply the changes.",
     "ui.package.hintRemoved": "This package will be removed when you apply the changes.",
     "ui.package.requiredTitle": "manually added",
-    "ui.package.requiredText": "This package is required in your composer.json but not installed.",
+    "ui.package.requiredText": "This package is required in your <code>composer.json</code> but not installed.",
```
Comment (aschempp) [inline src/i18n/en.json]: For security reasons, HTML cannot be included by default in labels.

**Comment by aschempp** [comment]
I have fixed the spelling mistake in the language files

File: src/components/App.vue (hunk 2 above)
Comment (xchs) [reply src/components/App.vue]: Because they're missing there:

cm

Of course, you could also add these spaces somewhere else.

File: src/i18n/en.json (hunk 3 above)
Comment (xchs) [reply src/i18n/en.json]: But there are already HTML tags in other labels as well.

Comment (aschempp) [reply src/i18n/en.json]: Yes, in very limited cases where we must support them…

File: src/components/App.vue (hunk 2 above)
Comment (aschempp) [reply src/components/App.vue]: There is a newline, which should be converted to spaces in HTML? 🤔

File: src/i18n/en.json (hunk 3 above)
Comment (xchs) [reply src/i18n/en.json]: I have seen it e.g. here:

https://github.com/xchs/contao-manager/blob/master/src/i18n/en.json#L151
https://github.com/xchs/contao-manager/blob/master/src/i18n/en.json#L158
